sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/util/SearchPagedFlux.java

```diff
@@ -101,4 +102,20 @@ public SearchPagedFlux(Supplier<Mono<SearchPagedResponse>> firstPageRetriever,
             });
     }
 
+    /**
+     * The answer results based on the search request.
+     * <p>
+     * If {@code answers} wasn't supplied in the request this will be null.
+     *
+     * @return The answer results if {@code answers} were supplied in the request, otherwise null.
+     */
+    public Mono<List<AnswerResult>> getAnswers() {
+        return metadataSupplier.get()
+            .flatMap(metaData -> {
+                if (metaData.getFirstPageResponse().getAnswers() == null) {
```

Comment:
Will all the results of `getAnswers()` always be on the first page? Also, if the user has used `byPage()` or simply subscribed to the paged flux, is it necessary to make another call to the service? Can we cache the answers from the first call that happened through other APIs?

Reply:
I think this is a good idea to make it so another API call isn't being made, mind filling an issue for this as this is pre-existing.